app/src/main/res/values/strings.xml

```diff
@@ -53,10 +53,10 @@
     <string name="nav_menu_graphs">Graphs</string>
     <string name="nav_menu_settings">Settings</string>
     <string name="nav_feedback">Feedback</string>
-    <string name="nav_menu_bitcoin_donation">Donate Bitcoin</string>
+    <string name="nav_menu_bitcoin_donation">Donate bitcoin</string>
 
     <!-- Toast messages -->
-    <string name="connection_not_found">Check your internet connection</string>
+    <string name="connection_not_found">Check your Internet connection</string>
```

Comment:
Could you please update localization by weblate?
For this specific translation - Internet is in general name of a specific internet. By my opinion then, there should be capital letter too.

Reply:
I didn't understand that, should it be "network connection"?